pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/forward/ForwardIndexType.java

```diff
@@ -125,6 +125,11 @@ public ForwardIndexConfig getDefaultConfig() {
     return ForwardIndexConfig.DEFAULT;
   }
 
+  @Override
+  public String getPrettyName() {
+    return "forward";
```

Comment:
Same here, can we please extract the constant before

Reply:
Will do.